Documentation/gettingstarted/grpc.rst

```diff
@@ -0,0 +1,297 @@
+*****************************
+Getting Started Securing gRPC 
+*****************************
+
+This document serves as an introduction to using Cilium to enforce gRPC-aware
+security policies.  It is a detailed walk-through of getting a single-node
+Cilium environment running on your machine. It is designed to take 15-30
+minutes.
+
+.. include:: gsg_intro.rst
+.. include:: minikube_intro.rst
+
+Step 1: Installing Cilium
+=========================
+
+The next step is to install Cilium into your Kubernetes cluster.  Cilium installation
+leverages the `Kubernetes Daemon Set <https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/>`_
+abstraction, which will deploy one Cilium pod per
+cluster node.   This Cilium pod will run in the ``kube-system`` namespace along with
+all other system relevant daemons and services.  The Cilium pod will run both the Cilium
+agent and the Cilium CNI plugin.
+
+Since gRPC runs over HTTP2, Cilium must be run with the Envoy proxy to properly forward
+and filter gRPC requests.   This is not yet the default, we will download and edit
+the standard cilium daemonset YAML. 
+
+Download Cilium, for example, using wget: 
+
+.. parsed-literal::
+
+    $ wget |SCM_WEB|\/examples/kubernetes/cilium.yaml
+
+Then make the following changes using a text editor.  
+
+First, change the Cilium container image field from ``cilium/cilium:stable`` to 
+``cilium/cilium:envoy``
+
+Second, add the ``--envoy-proxy`` flag to the args passed to the cilium-agent to indicate that 
```

Comment:
Format in code ```cilium-agent``` or text "Cilium Agent", to be consistent with other parts of our Docs.